projects/dsp-ui/src/lib/viewer/values/list-value/list-value.component.ts

```diff
@@ -43,7 +43,7 @@ export class ListValueComponent extends BaseValueComponent implements OnInit, On
 
    getInitValue(): string | null {
     if (this.displayValue !== undefined) {
-      return this.displayValue.listNodeLabel;
+      return this.displayValue.listNode;
```

Comment:
I think the node should be represented as an object like in the case of other complex values (interval, time, date) implementing `ControlValueAccessor` (see https://dasch.myjetbrains.com/youtrack/issue/DSP-208).

Otherwise the component has to maintain the state of two things: label and Iri of the node. If `ControlValueAccessor` is implemented, it will be handled solely by `FormControl`. I am also thinking about using generics to make that clearer (https://dasch.myjetbrains.com/youtrack/issue/DSP-236). 

This is a good article about `ControlValueAccessor `: https://indepth.dev/never-again-be-confused-when-implementing-controlvalueaccessor-in-angular-forms/

Reply:
Refactoring working components is not the purpose of this PR.

Reply:
> Refactoring working components is not the purpose of this PR.

But there is clearly a change in list-value.component.ts, so this seems out of scope too

Reply:
> But there is clearly a change in list-value.component.ts, so this seems out of scope too

The change I have applied here is for ErrorHandling that is the purpose of this PR

Reply:
`getInitValue()` was changed because the list component is not implemented properly (it has to maintain the state of two things instead of one that would be delegated to `FormControl`.). So this should be fixed. I did not say it has to happen in this PR. But then the change of `getInitValue()` shouldn't have happened here either.

Reply:
the change of getInitValue was necessary for the `standardValueComparisonFunc` function that is necessary for ErrorHandling!